02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -1,8 +1,82 @@
+The characteristics of the data produced by these two technologies can be quite different.
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Introduction to microarray technology](#introduction-to-microarray-technology)
+  - [Microarray data **strengths**:](#microarray-data-strengths)
+  - [Microarray data **limitations**:](#microarray-data-limitations)
+- [About quantile normalization](#about-quantile-normalization)
+- [More resources on microarray technology:](#more-resources-on-microarray-technology)
+  - [Why doesn't the gene I care about show up in this microarray dataset?](#why-doesnt-the-gene-i-care-about-show-up-in-this-microarray-dataset)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction to microarray technology
+
+Microarrays measure gene expression using chips filled with oligonucleotide probes designed to hybridize to labeled RNA samples. 
+After hybridization, the microarrays are scanned, and the fluorescence intensity for each probe is measured.
+The fluorescence intensity indicates the number of labeled fragments bound and therefore the relative quantity of the transcript the probe is designed for. 
+
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg" width=400>  
+
+[@microarray-video]
+
+There are many different kinds of microarray platforms. 
+
+Microarrays can be categorized by their probe chemistry and manufacturer: [Affymetrix GeneChips](https://www.thermofisher.com/us/en/home/life-science/microarray-analysis/transcriptome-profiling-microarrays.html), [Illumina Beadarray](https://www.illumina.com/documents/products/datasheets/datasheet_gene_exp_analysis.pdf); all of which supported on refine.bio.
+But, refine.bio only supports single-color arrays, although there are [two-color arrays](https://www.ebi.ac.uk/training/online/course/functional-genomics-ii-common-technologies-and-data-analysis-methods/microarrays), so our examples and advice are generally from the perspective of using single-color array. 
+
+As with all experimental methods, microarrays have strengths and limitations that you should consider in regards to your scientific questions. 
+
+### Microarray data **strengths**:  
+
+- Microarray is generally less expensive than RNA-seq - you can afford more replicates and get higher statistical power [@Tarca2006]. 
+- Microarray often has a faster turn-around than RNA-seq [@LCSciences2014].
```

Comment:
I think the costs thing definitely still applies (for now) but I can hedge it more if we must.

Reply:
Fair enough. How do we get across to people the idea that "microarray data is still relevant even with new tech around" (I couldn't find sources that say this for us). What wording makes sense that we can support?

Reply:
I've been struggling from the beginning to get the wording right for this point. I've tracked this issue: #296 and propose we move forward with this PR otherwise. Does that sound fine @jashapiro?